presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -1667,4 +1668,16 @@ public FeaturesConfig setPartialResultsMaxExecutionTimeMultiplier(double partial
         this.partialResultsMaxExecutionTimeMultiplier = partialResultsMaxExecutionTimeMultiplier;
         return this;
     }
+
+    @Config("table-snapshots-enabled")
```

Comment:
You might want to add this to `PrismConfig` unless we really want to implement table snapshot generically.